res/xml/qwerty.xml

```diff
@@ -10,18 +10,18 @@
     <key key0="u" key2="7" key3="&amp;"/>
     <key key0="i" key2="8" key3="*"/>
     <key key0="o" key1="accent_macron" key2="9" key3="(" key4=")"/>
-    <key key0="p" key2="0"/>
+    <key key0="p" key1="0"/>
   </row>
   <row>
-    <key shift="0.5" key0="a" key1="tab" key2="`"/>
+    <key shift="0.5" key0="a" key4="tab" key2="`"/>
```

Comment:
`tab` and `l` don't seem to be problematic for qwerty as they are not close to the edge of the screen.

Reply:
It's true, that row of keys has some extra margin. but I still felt like an improvement to not hit my finger on the sharp edge of my phone case in an extended and natural finger swipe.
I guess the new tab position I'll have to create some muscle memory first, because I was so used to the old. That is why it's good to test first. 
For the pipe symbol, the adoption of the new position was natural and seems right for me.